config/settings/test.yml

```diff
@@ -13,3 +13,8 @@ token_encrypt:
   salt: 'dGcTBhXrQjsvQkbDQCkAIdogYjzOiIQpjToNLjqyzslKShfTvxIJHmMZhzdNjOqtDZKtxutyHBZmNHKOcGMfHzxwCFeOTBeEHYHNedtrmdfLzHMTRSDJCRMURaAjRrDq'
 
 mailer_host: 'example.com'
+
+symphony_web_services:
+  # trailing slashes expected unless it's the end of the URI path
+  base_url: 'http://example.com/symws/'
+  curr_loc_path: 'v1/catalog/item/barcode/'
```

Comment:
my thinking here is that there will be other symphony web services paths we will be using in the future, such as the one for lookupTitleInfo ...